src/cart/utils/fault-inject-cart.yaml

```diff
@@ -37,3 +37,22 @@ fault_config:
     interval:     1
     max_faults:   2
     err_code:     1011
+
+    # SWIM messages drop fault injection
+    # probability_x should be 1 always to drop SWIM messages
+    # Change probability_y to non-zero to introduce random failures.
+    # probability_y: 20  - will introduce 5% of message loss
+    # probability_y: 100 - will introduce 1% of message loss
+  - id:            65025
+    interval:      0
+    max_faults:    0
+    probability_x: 1
+    probability_y: 0
+    err_code:      0
+    # Comma separated list of SWIM messages drop fault injection arguments:
+    # rank=<num> - Set it to the rank number which should be evicted by
+    # simulation of network failures.
+    # delay=<num> - Set it to the number of seconds on which fault injection
+    # will be delayed. It can be useful to complete an initialization and
+    # creation of pool/container need for testing.
+    argument:     "rank=1,delay=10"
```

Comment:
I don’t know if these suggestions makes sense.

Is there any chance to support more than one rank for drop injection?
I tried using `"rank=1,rank=2,delay=300"` and it seems that it takes only the later rank value. I mean only rank 2 is affected

`*** SWIM id=2 should fail`

May we have a “duration” parameter? So, we can create transient failures.

Reply:
Currently only single rank is supported. Next instance with the same name will overwrite previous one.

I can add  “duration” parameter but is it really make sence for testing? It will not be restarted after duration complete.

Reply:
We can already create transient failures can't we?

Reply:
Actually we already have "max_faults" for this. It will stop injection after this count fails.

Reply:
I was trying to understand if we have all the required parameters to reproduce the experiments on the paper pointed out by Liang. It seems that we only need to use the same number of nodes, message loss rate and run the experiments for 30 minutes.
To me SWIM is a new topic. So, after all. No new parameters are required. Sorry for the confusion.

Reply:
This patch already contain all functionality for proposed experiments. This patch is for landing to master for future experiments and testing.

Reply:
@berserk-fury could you approve it please if it works for you?